root/usr/share/clearwater-infrastructure/scripts/restart/chronos_restart

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+# WARNING - NON-STANDARD BEHAVOUR INCOMING
+#
+# Contrary to its name and position in the filesystem, this script does NOT
+# restart chronos.  Instead it considers if the chronos shared configuration
+# should be updated based on the current global shared configuration and, if
+# so, uploads the new configuration to etcd (which will later restart the
+# chronos instance after queue manager has done its thing).
+
+IFS=,
+CHRONOS_CONF=/etc/chronos/chronos_shared.conf
+
+# Read the up-to-date global configuration
+. /etc/clearwater/config
+
+# Generate the related Chronos configuration
+TMPFILE=$(mktemp)
+echo "[dns]" > $TMPFILE
+for server in $signaling_dns_server
+do
+  echo servers = $server >> $TMPFILE
+done
+
+echo >> $TMPFILE
+echo "[sites]" >> $TMPFILE
```

Comment:
Should we check for multiple sites, and only include this section in that scenario?